cranelift/codegen/src/ir/function.rs

```diff
@@ -255,6 +255,11 @@ impl FunctionStencil {
         self.get_dyn_scale(dyn_ty)
     }
 
+    /// Find the data for the given stack slot
+    pub fn get_stack_slot_data(&self, stack_slot: StackSlot) -> &StackSlotData {
+        &self.sized_stack_slots[stack_slot]
+    }
```

Comment:
This field is already public.

Reply:
I'll remove this in favor of using the pub field directly; probably just a miss on the visibility when the wasmfx contributors originally authored these changes (looks like field has been pub for a long time).